web/concrete/src/Asset/JavascriptAsset.php

```diff
@@ -1,80 +1,95 @@
 <?php
 namespace Concrete\Core\Asset;
+
 use HtmlObject\Element;
 use Config;
 
-class JavascriptAsset extends Asset {
+class JavascriptAsset extends Asset
+{
+
+    protected $assetSupportsMinification = true;
+    protected $assetSupportsCombination = true;
 
-	protected $assetSupportsMinification = true;
-	protected $assetSupportsCombination = true;
+    protected $scriptType = 'text/javascript';
```

Comment:
Well, I kind of see what you are trying to say here, I could have made this in a may that the TagAttributes include the "type" attribute as well (here again that would be quite a change), that would make sense, but that would quite change everything that is done … 

Not sure this is a good idea there, (is it?)